diff -Nur python-werkzeug_0.1/setup.py python-werkzeug_0.1.new/setup.py
--- python-werkzeug_0.1/setup.py	2007-07-20 19:58:50.000000000 +0100
+++ python-werkzeug_0.1.new/setup.py	2007-07-20 20:58:46.000000000 +0100
@@ -1,9 +1,7 @@
 # -*- coding: utf-8 -*-
 import werkzeug
 import os
-import ez_setup
 from inspect import getdoc
-ez_setup.use_setuptools()
 
 from setuptools import setup, Extension, Feature
 
